its/Models/DotnetFrameworkV3TestCase.cs

```diff
@@ -24,8 +24,8 @@ public class DotnetFrameworkV3TestCase : BuildTestCase
 	{
 		public DotnetFrameworkV3TestCase()
 		{
-			Coverage = 25.0;
-			NcLocs = 49;
+			Coverage = 2.7;
+			NcLocs = 262;
```

Comment:
The .NET scanner bump changed these values

(another benefit to have them in-code rather than in variables is that we can change them here and not break master build 😛)